<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QVCRWidget</class>
 <widget class="QWidget" name="QVCRWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>228</width>
    <height>32</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Play Controls</string>
  </property>
  <layout class="QHBoxLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QPushButton" name="firstButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>First Frame</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Icons/FamFamFamIcons.qrc">
       <normaloff>:/icons/control_start_blue.png</normaloff>:/icons/control_start_blue.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="backButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Previous Frame</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Icons/FamFamFamIcons.qrc">
       <normaloff>:/icons/control_rewind_blue.png</normaloff>:/icons/control_rewind_blue.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="playButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>3</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Play</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Icons/FamFamFamIcons.qrc">
       <normaloff>:/icons/control_play_blue.png</normaloff>:/icons/control_play_blue.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pauseButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>2</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Pause</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Icons/FamFamFamIcons.qrc">
       <normaloff>:/icons/control_pause_blue.png</normaloff>:/icons/control_pause_blue.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="forwardButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Next Frame</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Icons/FamFamFamIcons.qrc">
       <normaloff>:/icons/control_fastforward_blue.png</normaloff>:/icons/control_fastforward_blue.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="lastButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Last Frame</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Icons/FamFamFamIcons.qrc">
       <normaloff>:/icons/control_end_blue.png</normaloff>:/icons/control_end_blue.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Icons/FamFamFamIcons.qrc"/>
  <include location="../../Applications/Icons/vtkSNLIcons.qrc"/>
 </resources>
 <connections/>
</ui>
